src/plugins/dataflow-tool/components/dataflow-program.tsx

```diff
@@ -99,6 +99,7 @@ interface IProps extends SizeMeProps {
   programIsRunning?: string;
   onCheckProgramRunState: (endTime: number) => void;
   tileHeight?: number;
+  programId?: number;
```

Comment:
Is `programId` used for anything?

Reply:
@tealefristoe You are absolutely correct.  I've taken it out.  I originally set up that `programId` up because I didn't quite understand all the architecture already available to me.  You can see how I was originally using it in [this commit](https://github.com/concord-consortium/collaborative-learning/commit/5b269f7e49836d9d798884769a938bbbd11aa9ec). 

Then, knowing it was a hack and I must be missing something, I eventually figured out I could leverage the built in stuff by registering the method, in [this commit](https://github.com/concord-consortium/collaborative-learning/commit/390e6f964527ba24350b47966588355f8a51dc1e). 

By the time I wrote the PR all that was a distant memory and I wrote the PR assuming that `programId` must have been doing something.  Anyway, thanks for catching that.  It's just as well as there is another `programId` being used in another context anyway.

Reply:
Noting for myself here that ultimately, it's `/src/models/document/document-content.ts`'s `getUniqueTitle` that is responsible for creating the default title.